src/CanvasInput/index.js

```diff
@@ -170,24 +145,22 @@ export default class CanvasInput extends React.Component {
   }
 
   handleMouseLeave = () => {
-    this.props.onUpdate({
-      action: 'mouseLeave',
-    })
+    this.props.onUpdate({action: 'mouseLeave'})
     this.setState({
-      mouse: undefined,
-      hoverRenderData: undefined,
-      hoverData: undefined,
-      layerProps: undefined,
+      mouse: null,
```

Comment:
Let's also initialize the state to these values (rather than an empty object).

Reply:
Done!